lang/typescript/typescript.py

```diff
@@ -18,6 +18,16 @@
 
```

Comment:
Don't we want `async` in typescript?

Reply:
Also javascript, but I think I'll leave that for another PR. If somebody wants to add it they'll probably need to implement a version of `code_modified_function` that does the construction of the function from its modifiers rather than use my default delegation implementation.